hyperopt/pyll/base.py

```diff
@@ -368,7 +368,7 @@ def arg(self):
         if extra_args_ok:
             # XXX: THIS IS NOT BEING TESTED AND IS OBVIOUSLY BROKEN
             # TODO: 'args' does not even exist at this point
-            binding[args_param].extend(args[code.co_argcount :])
+            binding[args_param].extend(args[code.co_argcount :])  # noqa: F821
```

Comment:
why you need the noqa here?

Reply:
Because like the comment on the lines above say, `args` is not defined.

Reply:
I guess this is working because some black magic with the pyll? or is it directly wrong?

Reply:
I have never heard of pyll so I could not say.  I am not a fan of magic in software engineering.

Reply:
nvm